src/styles/components/_sign-up-success-form.scss

```
@@ -0,0 +1,42 @@
+@import '../abstracts/_abstracts-dir';
+
+.sign-up-success-form {
+  display: flex;
+  flex-direction: column;
+  background-color: $background;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    padding: rem(16px 0);
+    margin-bottom: rem(24px);
+  }
+
+  &__header {
+    display: flex;
+    justify-content: center;
+    margin-bottom: rem(24px);
+  }
+
+  &__footer {
+    margin-bottom: rem(24px);
+  }
+
+  &__actionable {
+    display: flex;
+    justify-content: center;
+  }
+
```

How about a little rework on here? 🤓

```suggestion
.sign-up-success-form {
  display: flex;
  flex-direction: column;
  background-color: $background;
  
  &__image,
  &__header,
  &__actionable {
    display: flex;
    justify-content: center;
    
    &:not(:last-child) {
      margin-bottom: rem(24px);
    }
  }

  &__image {
    padding: rem(16px 0);
    margin-bottom: rem(24px);
  }
}
```